asf_search/CMR/translate.py

```diff
@@ -217,10 +217,12 @@ def translate_product(item: dict) -> dict:
         }
         properties['burst'] = burst
         properties['sceneName'] = properties['fileID']
+        properties['bytes'] = cast(int, get(umm, 'AdditionalAttributes', ('Name', 'BYTE_LENGTH'),  'Values', 0))
+
```

Comment:
Potential exception here if the field is poorly populated (rare, but has happened in the past)

Reply:
Thankfully the get method in asf-search can handle missing fields (defaulting them to `None`).